src/main/java/no/nav/pto/veilarbportefolje/hendelsesfilter/HendelseService.kt

```diff
@@ -0,0 +1,96 @@
+package no.nav.pto.veilarbportefolje.hendelsesfilter
+
+import no.nav.pto.veilarbportefolje.kafka.KafkaCommonKeyedConsumerService
+import no.nav.pto.veilarbportefolje.kafka.KafkaConfigCommon.Topic
+import no.nav.pto.veilarbportefolje.persononinfo.PdlIdentRepository
+import org.jetbrains.annotations.TestOnly
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import java.util.*
+
+/**
+ * Håndterer behandling av Kafka-meldinger fra [Topic.PORTEFOLJE_HENDELSESFILTER].
+ *
+ * Topic-et er et generisk topic som andre team kan produsere generelle oppfølgingshendelser på.
+ * Bruksområdet for disse hendelsene er i hovedsak å populere statusfiltre i Oversikten (veilarbportefoljeflatefs).
+ * Et eksempel på en hendelse er "Utgått varsel" som i skrivende stund produseres av "veilarbdialog"-applikasjonen.
+ *
+ * Denne klassen håndterer således funksjonalitet knyttet til å starte (les: lagre), oppdatere og stoppe (les: slette)
+ * hendelser.
+ */
+@Service
+class HendelseService(
+    @Autowired private val hendelseRepository: HendelseRepository,
+    @Autowired private val pdlIdentRepository: PdlIdentRepository
+) : KafkaCommonKeyedConsumerService<HendelseRecordValue>() {
+    private val logger: Logger = LoggerFactory.getLogger(HendelseService::class.java)
+
+    /**
+     * Behandle en [HendelseRecordValue] og tilhørende `hendelseId`:
+     *
+     * * dersom `hendelseRecordValue.operasjon` = [Operasjon.START] vil hendelsen kombineres med ID-en og persisteres
```

Comment:
```suggestion
     * * dersom `hendelseRecordValue.operasjon` = [Operasjon.START] vil hendelsen kombineres med ID-en og lagres
```

Kan vi bruke "lagre" i staden for "persistere"? Litt lettare å lese, men kanskje ikkje like teknisk korrekt?